sm/download/src/download.rs

```diff
@@ -71,16 +79,24 @@ impl Downloader {
             }
         })
         .await?;
+        let mut response = response;
```

Comment:
Why not simply make the response `mut` when first declared?

Reply:
fixed